config/locales/en.yml

```diff
@@ -1,4 +1,8 @@
 en:
+  time:
+    formats:
+      default: '%b %d, %Y %H:%M'
```

Comment:
Nice! I would suggest using 12-hour format instead of 24-hour though. I personally prefer 24-hour but it makes some people flip out!